renderdoc/os/posix/posix_libentry.cpp

```diff
@@ -42,9 +42,8 @@ void library_loaded()
   string curfile;
   FileIO::GetExecutableFilename(curfile);
 
-  if(curfile.find("/renderdoccmd") != string::npos ||
-     curfile.find("/renderdocui") != string::npos || curfile.find("/qrenderdoc") != string::npos ||
-     curfile.find("/system/bin/app_process") != string::npos)
+  if(curfile.find("renderdoccmd") != string::npos || curfile.find("/renderdocui") != string::npos ||
```

Comment:
Can you leave this as a check to `/renderdoccmd` and add a check for `.renderdoccmd` or `org.renderdoc.renderdoccmd`? As it is, this could (in theory) match against a substring of another executable. This seems stupid but it's happened before with things like `renderdoc bug repro.exe` so I'd like to make it safe :).